### DISCUSSION TITLE
is there a shortcut key help popup like settings

### DISCUSSION THREAD

**Discussion by ldijkman** [ajaxorg/ace#5302] [Q&A]
hello 

is there 
a shortcut key help popup 
like settings popup

is it command pallete F1   that is not working for me chrome popups something of chrome
ok F1 works now
but
```
 <input type="button" value="Settings" onClick='ace.edit("editor").execCommand("showSettingsMenu");' title="show settings menu">
<!-- not working -->
<input type="button" value="ShortCuts use F1 Key" onClick='ace.edit("editor").execCommand("showKeyboardShortcuts")' title="not working do not now the commad to open it">
<!-- not working -->
```

greet luberth
ACE editor https://youtu.be/lZW8KM6svH8?si=EMcXvzL5DZrvBU76
HTML preview visual scheduler touch friendly 
JumpMarks ScrollMarks BookMarks 
Breakpoints move as code is edited

https://codepen.io/ldijkman/pen/poqbbqa

![2023-09-03-060039_1920x1080_scrot](https://github.com/ajaxorg/ace/assets/45427770/d0e1d4d9-59f7-42c5-a88a-1fd8feff8f15)

https://codepen.io/ldijkman/pen/poqbbqa

**Comment by ldijkman** [comment]
ok got shortcut info popup
 working now from menu button

https://codepen.io/ldijkman/pen/poqbbqa

![2023-09-03-112507_1920x1080_scrot](https://github.com/ajaxorg/ace/assets/45427770/cecc2621-3672-4438-8c30-dcf5b4aff191)

**Reply by nightwing** [reply]
the command palette (which can be opened by F1 key), is probably a better alternative, as it lists all commands in a more readable way and also allows to filter and invoke them.

**Reply by ldijkman** [reply]
Thanks
yes but F1 needs focus on editor if not chrome opens F1
but both work now
https://codepen.io/ldijkman/pen/poqbbqa

2 screenshots hmmm second screenshot needed scrot from terminal

![2023-09-03-114835_1920x1080_scrot](https://github.com/ajaxorg/ace/assets/45427770/e49f29cd-4caa-4c51-9338-3450c5418899)

![2023-09-03-115414_1920x1080_scrot](https://github.com/ajaxorg/ace/assets/45427770/729f5da5-5189-4a28-afd1-aedf9e495f5e)

**Comment by ldijkman** [comment]
is it possible 
to open the F1 popup
from a menu button?

what would be the javascript command?

![2023-09-03-115414_1920x1080_scrot](https://github.com/ajaxorg/ace/assets/45427770/729f5da5-5189-4a28-afd1-aedf9e495f5e)

**Reply by nightwing** [reply]
you can use `editor.execCommand("openCommandPallete")`

**Reply by ldijkman** [reply]
Thank you very much!
it works

```
<input type="button" value="F1 Key" onClick='editor.execCommand("openCommandPallete")' title="show F1 key shortcut keys popup">
<input type="button" value="ShortCut Keys" onClick=' editor.execCommand("showKeyboardShortcuts")' title="show shortcut keys popup">
```

https://plnkr.co/edit/Digj5Fw7D6hxaTU4

![2023-09-03-131334_1920x1080_scrot](https://github.com/ajaxorg/ace/assets/45427770/09f17bf3-db77-4473-945a-60e26f968e88)
